tests/libdrgn.py

```diff
@@ -203,10 +208,12 @@ def __repr__(self):
 
 
 class Lexer:
-    def __init__(self, func, str):
-        self._lexer = _drgn_lexer()
+    def __init__(self, func, str, cpp=False):
+        self._c_family_lexer = _drgn_c_family_lexer()
+        self._lexer = self._c_family_lexer.__getattribute__("lexer")
```

Comment:
Is there some reason that getting the attribute directly doesn't work? I.e.

```suggestion
        self._lexer = self._c_family_lexer.lexer
```